pinc/CharacterSelector.inc

```diff
@@ -6,10 +6,19 @@ class CharacterSelector
     {
         global $utf8_site;
 
+        $mru =
+        [
+            "code" => "&#x1F550;",
```

Comment:
Wow, the clock is actually rendered rather large in Chrome on Mac which is unexpected (and, IMHO, unwelcome). That's a little frustrating.

Reply:
This is unfortunate.